Bigtable/src/BigtableClient.php

```diff
@@ -136,24 +136,23 @@ public function __construct(array $config = [])
      * ```
      *
      * @codingStandardsIgnoreStart
-     * @see https://cloud.google.com/bigtable/docs/reference/admin/rpc/google.bigtable.admin.v2#CreateInstanceRequest CreateInstanceRequest
+     * @see https://cloud.google.com/bigtable/docs/reference/admin/rpc/google.bigtable.admin.v2#google.bigtable.admin.v2.CreateInstanceRequest CreateInstanceRequest
      *
      * @param string $instanceId The instance ID.
      * @param array $options [optional] {
      *     Configuration options
      *
      *     @type string $displayName **Defaults to** the value of $instanceId.
-     *     @type int $instanceType  Possible values include `Instance_Type::PRODUCTION` and `Instance_Type::DEVELOPMENT`.
-     *           **Defaults to**`Instance_Type::TYPE_UNSPECIFIED`.
-     *     @type array $labels For more information, see
+     *     @type int $instanceType Possible values include @var Instance::INSTANCE_TYPE_PRODUCTION  and @var Instance::INSTANCE_TYPE_DEVELOPMENT.
+     *           **Defaults to** @var Instance::INSTANCE_TYPE_UNSPECIFIED.
+     *     @type array $labels as key/value pair ['foo' => 'bar']. For more information, see
      *           [Using labels to organize Google Cloud Platform resources](https://cloudplatform.googleblog.com/2015/10/using-labels-to-organize-Google-Cloud-Platform-resources.html).
-     *     @type array $clusters [] {
-     *           array {
-     *                 string $clusterId
-     *                 string $locationId
-     *                 int $serveNodes
-     *                 int $storageType The storage media type for persisting Bigtable data. Possible values include `Instance::STORAGE_TYPE_SSD` and `Instance::STORAGE_TYPE_HDD`.
-     *                 **Defaults to** `Instance::STORAGE_TYPE_UNSPECIFIED`.
+     *     @type Cluster[] $clusters {
```

Comment:
I'm a bit confused by this, `Cluster[]` tells me this will be an array of Cluster objects - yet the following documentation (with clusterId, locationId, etc.) references this as an array. Which are you planning to implement?

Reply:
This is an array in the form of
$options = [    
    'displayName' => 'My Instance',
    'instanceType' => Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_DEVELOPMENT,
    'labels' => ['foo' => 'bar'],
    'clusters' => [
        [
            'clusterId' => $clusterId,
            'locationId'=> $locationId,
            'nodeCount' => 4,
            'storageType' => StorageType::SSD
        ],
        [
            'clusterId' => $clusterId2,
            'locationId'=> $locationId2,
            'nodeCount' => 2,
            'storageType' => StorageType::SSD
        ]
    ]
];
$bigtable->createInstance($instanceId, $options);

Reply:
If you're gonna pass ['clusterId' => ..., 'locationId' => ...], shouldn't your PHPDoc specifies a normal array?
```php
@type array $clusters {
}
```

seems more accurate.

From your current PHPDoc, it looks like you're gonna pass this:

```php
$options = [
    'displayName' => 'My Instance',
    'instanceType' => Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_DEVELOPMENT,
    'labels' => ['foo' => 'bar'],
    'clusters' => [
       new Cluster(....), new Cluster(...)
    ]
];
$bigtable->createInstance('my-instance', $options);
```